package-lock.json

```diff
@@ -10895,7 +10895,7 @@
       "dev": true
     },
     "tar": {
-      "version": "2.2.1",
+      "version": "4.4.2",
       "resolved": "https://registry.npmjs.org/tar/-/tar-2.2.1.tgz",
```

Comment:
its still resolved to 2.2.1... Did you test this?

Reply:
was about to comment on that, i did npm install and did npm audit, doesn't seem to change the 2.2.1 to 4.4.2, but the vulnerability doesnt show up, let me check once again.